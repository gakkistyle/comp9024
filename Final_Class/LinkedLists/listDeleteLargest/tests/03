9 1 8 2 7 4
