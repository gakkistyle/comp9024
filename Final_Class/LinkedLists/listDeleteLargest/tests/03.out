Enter list: 
Original list: [9] -> [1] -> [8] -> [2] -> [7] -> [4] -> X
After deleting largest: [1] -> [8] -> [2] -> [7] -> [4] -> X
The largest value was: 9
